layouts/_default/resource.html

```diff
@@ -0,0 +1,34 @@
+<section>
+    <a class="block" href="{{ .Permalink }}">
+        <img src="{{ .Params.image | absURL }}" alt="" class="rounded-t-xl object-cover" style="height:212px; width:100%">
+    </a>
+    <div class="bg-[#ECF6FF] rounded-b-xl">
+        <section class="px-6 pt-4">
+            <p class="mt-3 pb-3 text-base text-[#A32C00] font-semibold">{{ .Params.event_date }}</p>
+            <h3 class="text-xl font-extrabold pb-3">
+                <a class="block" href="{{ .Permalink }}">{{ .Params.name }}</a>
+            </h3>
+            <ul class="mt-3 pb-6">
```

Comment:
Tags for `categories`, `events`, and `topics` will all appear on one line. I considered using different colors to differentiate the taxonomy type, but it wouldn't be immediately obvious to the user. I thought having them together instead of separate lines would look better if we get a little tag heavy for a resource.